lib/Jsrt/Core/JsrtContextCore.h

```diff
@@ -163,7 +163,6 @@ class ChakraCoreHostScriptContext sealed : public HostScriptContext
 
     HRESULT EnqueuePromiseTask(Js::Var taskVar) override
     {
-        AssertMsg(false, "jsrt should have set the promise callback");
```

Comment:
That's exactly what happened, NativeTests.exe crashes in debug mode with that assert in. In the JSRT overview in the wiki, there is mention that you need to use this function in order to use promises. Should something be added to the JsSetPromiseContinuationCallback documentation as well?